bin/build.sh

```diff
@@ -2,17 +2,46 @@
 
 wca_url="https://worldcubeassociation.org/"
 
-# Find all Markdown files and build PDFs out of them.
+# Find formal and legal Markdown files and build PDFs out of them.
 find 'documents' -name '*.md' | while read file; do
+  echo "Processing $file, vanilla style"
```

Comment:
Do we want log messages? I found them extremely helpful during development, and with multiple folder structures I think they would be beneficial during deployment as well.

Reply:
I definitely don't mind having that, the travis log is pretty short anyway.